home page working

devsoob opened the pull request.

synuns commented on package.json:
```diff
@@ -9,12 +9,19 @@
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.5.0",
+    "react-slick": "^0.29.0",
```
commit title에는 fontawesome만 적혀있는데 react-slick과 slick-carousel이 추가되어 있네요.
추가한 디펜던시를 확실하게 명시해주세요.

synuns commented on package.json:
```diff
@@ -31,6 +38,7 @@
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "prettier": "^2.8.1",
+    "slick-carousel": "^1.8.1",
```
commit title에는 fontawesome만 적혀있는데 react-slick과 slick-carousel이 추가되어 있네요.
추가한 디펜던시를 확실하게 명시해주세요.

synuns commented on .eslintrc.cjs:
```diff
@@ -33,5 +33,6 @@ module.exports = {
     'react/prop-types': 0, // prop-types를 선언해주어야하는 규칙 해제
     'no-param-reassign': 0, // 파라미터 변경을 허용하지 않는 규칙 해제
     'no-alert': 0, // alert 제한 규칙 해제
+    'import/core-modules': [electron], // slider devDependencies 규칙 해제
```
eslint 수정에 대한 커밋은 따로 생성해주세요

synuns replied:
이전 커밋이 Add dependency fontawesome으로 같은 타이틀의 커밋이네요.
하나로 합쳐서 커밋하면 좋을 것 같군요

synuns commented on src/components/home/House.jsx:
```diff
@@ -0,0 +1,48 @@
+import { faHouseChimney } from '@fortawesome/free-solid-svg-icons';
+import styled from 'styled-components';
+
+function House() {
+  return (
+    <HouseBox>
+      <HouseList>
+        <Link to="/details/:1">
+          <FontAwesomeIcon icon={faHouseChimney} className="house" />
+        </Link>
+      </HouseList>
+      <HouseList>
+        <Link to="/details/:2">
+          <FontAwesomeIcon icon={faHouseChimney} className="house" />
+        </Link>
+      </HouseList>
+      <HouseList>
+        <Link to="/details/:3">
+          <FontAwesomeIcon icon={faHouseChimney} className="house" />
+        </Link>
+      </HouseList>
+    </HouseBox>
+  );
+}
```
house 컴포넌트 내부에 houselist가 존재하는 관계가 모순적인 것 같습니다.
이름을 반대로 바꾸거나 직관적으로 이해하기 쉽게 변경해주세요

devsoob committed:
➕ Add dependency fontawesome, react-slick, slick-carousel

devsoob committed:
🔧 Update eslint rules
slider devDependencies 규칙 해제

devsoob committed:
💄 Add House component, Slider component, Home page
- HouseList component 추가(집 모양 아이콘이 하나의 게시글)
- 서울시 각 구별 게시글들을 볼 수 있는 Slider component 추가
- Home 페이지에  Slider import

devsoob commented:
수정사항 반영해서 변경했습니다. 확인부탁드립니다.

synuns committed:
Merge branch 'dev' into feature/home
